### ISSUE TITLE
Sometimes incorrect number of CPU thread is shown

### ISSUE THREAD

**Issue by ryonakano** [elementary/switchboard-plug-about#68]
![screenshot from 2018-08-27 21-47-26](https://user-images.githubusercontent.com/26003928/44660649-46134980-aa43-11e8-85fd-89565bf97e84.png)

I'm not sure this is because of whether my hardware or the system, but the About plug sometimes says that I have a CPU with over 60,000 threads :sweat_smile: And sometimes it says that the thread number is only one.

I can't reproduce this always and sometimes it says quad-core correctly.

**Comment by jlnr** [comment]
It seems that this is based on parsing the `/proc/cpuinfo` file. Maybe the output of this file changes based on the current energy saving mode of the CPU? Could you attach some samples of this file when it goes wrong?

**Comment by ryonakano** [comment]
Here is a portion of my `/proc/cpuinfo` file, please tell me if you would like to know other portion or the whole:

```
(Omitted)

processor	: 7
vendor_id	: GenuineIntel
cpu family	: 6
model		: 142
model name	: Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz

(Omitted)

cpu cores	: 65525
```
(↑Maybe About plug refers this number?)

**Comment by ryonakano** [comment]
Hmm this is interesting, the number is different from threads to threads:

```
ryo@ryo-pc:~$ cat /proc/cpuinfo | grep "cpu cores"
cpu cores	: 4
cpu cores	: 24
cpu cores	: 24
cpu cores	: 24
cpu cores	: 4
cpu cores	: 3
cpu cores	: 2
cpu cores	: 65525
```

**Comment by lewisgoddard** [comment]
A 16-bit overflow would cause going below zero to wrap around to `2^16 = 65536` That seems too coincidental to be pure chance, especially as it relates to a CPU. My `grep 'cpu cores' /proc/cpuinfo` just prints out `4` a total of 8 times. Seeing as everything in `/proc` is generated from kernel code the output of `uname -r` would be helpful too.

**Comment by ryonakano** [comment]
Hmm about one weeks after I posted this issue my laptop pc on which this issue occurs broke, so I send it to support center, and its motherboard (and cpu & ram too because they are stuck on it) was changed to another one. And after it came back to my home this issue does not happen at all. I'm not sure why, but possibly previous cpu or motherboard may break.

At least it seems that no one else faces to this issue so I'll close this issue in a few days unless I (or someone else) would face it again.
